ethereum/llama/aave_treasury/aave_tokens.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS llama;
+
+CREATE TABLE IF NOT EXISTS llama.aave_tokens (   
+    token_address bytea PRIMARY KEY,
+    decimals int4,
+    symbol text,
+    erc20_address bytea,
```

Comment:
Is this the same as `underlying_token`?

Reply:
conceptually, yes. Maybe that's a better name.